python/cudf/cudf/_lib/cpp/hash.pxd

```diff
@@ -7,40 +7,52 @@ from libcpp.vector cimport vector
 from cudf._lib.cpp.column.column cimport column
 from cudf._lib.cpp.table.table cimport table
 from cudf._lib.cpp.table.table_view cimport table_view
+from cudf._lib.exception_handler cimport cudf_exception_handler
 
 
 cdef extern from "cudf/hashing.hpp" namespace "cudf::hashing" nogil:
 
     cdef unique_ptr[column] murmurhash3_x86_32 "cudf::hashing::murmurhash3_x86_32" (
         const table_view& input,
         const uint32_t seed
-    ) except +
+    ) except +cudf_exception_handler
+
+    cdef unique_ptr[table] murmurhash3_x64_128 "cudf::hashing::murmurhash3_x64_128" (
```

Comment:
I suspect all of these are redundant in this file. Could we try removing them and see if things still compile?
```suggestion
    cdef unique_ptr[table] murmurhash3_x64_128
```